test/functional/test_site/requirements/createModal.md

```diff
@@ -0,0 +1,5 @@
+<!-- Used in test_site/testIncludeMultipleModals.md---->
```

Comment:
`createModal.md` sounds a little misleading here, since this isn't a function that creates a modal or anything like that. Shall we just name this `testModal.md`?

Reply:
Okay sure